### DISCUSSION TITLE
Arduino Cloud doesn't show Lion:Bit S3 STEM Dev Board

### DISCUSSION THREAD

**Discussion by PradeepKotu** [espressif/arduino-esp32#9926] [General]
Arduino Cloud doesn't show Lion:Bit S3 STEM Dev Board. It was added to arduino-esp32 last year. Still not shows in https://app.arduino.cc/things what we need to do for this issue. 
<img width="601" alt="Untitled" src="https://github.com/espressif/arduino-esp32/assets/36191520/4fc9e9ea-2740-4254-a902-11b75daa3913">

**Comment by me-no-dev** [comment]
cc @pillo79

**Reply by PradeepKotu** [reply]
Do we need to do request or does it take some time?

**Reply by me-no-dev** [reply]
I do not know how are the features organized in Arduino cloud, so I linked @pillo79 to maybe clarify. It's definitely not pulling our latest core

**Reply by pillo79** [reply]
Sorry, was unavailable for a few days. Definitely not my area as well :slightly_smiling_face: but thanks for the ping. I have notified the Cloud team of this issue.

**Reply by PradeepKotu** [reply]
Can we take someone's  help for this issue?.
